lib/src/git_backend.rs

```diff
@@ -1248,8 +1248,30 @@ fn write_tree_conflict(
         filename: name.into(),
         oid: tree_id.as_bytes().try_into().unwrap(),
     })
-    .sorted_unstable()
-    .collect();
+    .collect_vec();
+    let readme_id = repo
+        .write_blob(
+            r#"This is the contents if a jj commit with conflicts.
```

Comment:
Perhaps say something like:

> "This commit was made by `jj`, https://github.com/martinvonz/jj. The commit contains file conflicts, and therefore looks wrong when used with plain Git or tools that are familiar with `jj`."

This would work even if we exported such commits to GitHub.

Reply:
Sounds good. Thanks